Makefile

```diff
@@ -119,23 +144,19 @@ generate: install-mockgen
 
 format: install-goimports
 	@echo "> Formating code..."
-	test -z $$(goimports -l -w -local github.com/buildpacks/lifecycle .)
-
-verify-jq:
-ifeq (, $(shell which jq))
-	$(error "No jq in $$PATH, please install jq")
-endif
+	$(if $(shell goimports -l -w -local github.com/buildpacks/lifecycle .), @echo Fixed formatting errors. Re-run && exit 1)
 
 test: unit acceptance
 
-unit: verify-jq format lint install-yj
+unit: UNIT_PACKAGES=$(shell $(GOCMD) list ./... | grep -v acceptance)
+unit: format lint install-yj
 	@echo "> Running unit tests..."
-	$(GOTEST) -v -count=1 $$($(GOCMD) list ./... | grep -v acceptance)
+	$(GOTEST) -v -count=1 $(UNIT_PACKAGES)
```

Comment:
Had to refactor out the inlined subcommand since the syntax doesn't work with Windows and Linux